docs/setup_guides/hpc.rst

```diff
@@ -117,12 +125,20 @@ securing Jupyter, check out
 `Securing a notebook server <http://jupyter-notebook.readthedocs.io/en/stable/public_server.html#securing-a-notebook-server>`__
 in the Jupyter documentation.
 
+Finally, we may want to configure dask's dashboard to forward through Jupyter.
+This can be done by editing the dask distributed config file, e.g.:
+``.config/dask/distributed.yaml``. In this file, set:
+
+.. code:: python
+
+    diagnostics-link: "/proxy/{port}/status"
```

Comment:
Was not sure where to add this, I added it beyond distributed at first, but it was needed at the root of the yaml file.

Reply:
We should also maybe add a full link to where the dashboard is expected to be find, e.g. https://jupyter-server/user/username/proxy/8787/status if I understand correctly.

Reply:
I actually think this still needs to be a relative link, something like `../proxy/{port}/status`. @mrocklin had suggested this absolute path but I don't think this is working for folks in practice. The differences in the url from the standard jupyter notebook interface and jupyter lab interface further complicates this.

Reply:
Neither `../proxy` nor `/proxy` works for me with jupyterhub or manually launched notebook, the same behavior can be observed:
URL for `/proxy`:
````
https://jupyterhub.sis.cnes.fr/user/eynardbg/files/proxy/38238/status
````

URL for `../proxy`:
````
http://node136.sis.cnes.fr:8877/files/git/code-snippets/proxy/46876/status
````

So it always redirect to `notebook/files/`, and then absolute or relative path from where the notebook is located...

Reply:
On standard jupyter notebook, solutions that work for me either from manually launched notebook or Jupyterhub (see https://github.com/dask/dask-jobqueue/pull/144/files) are:
````
distributed.dashboard.link: "/proxy/{port}/status" # for user launched notebook
distributed.dashboard.link: "/user/{JUPYTERHUB_USER}/proxy/{port}/status" # for jupyterhub launched notebook
````

Reply:
Given the instructions in 216 ("Now open http://localhost:8888 on your local machine"), I think it's safe to recommend

```
diagnostics-link: "http://localhost:8888/proxy/{port}/status"
```

That's how we set it up in the [jobqueue.yaml on NCAR/pangeo](https://github.com/NCAR/pangeo/blob/master/server-side/jobqueue.yaml)